### PR TITLE
Added API call to fetch details of a bug by bug ID.

### PR DESCRIPTION
**Pull request by rtnpro**
This is a very crude version. For some reason, the `longtext` `description` field in `bug` table in db is not getting stored in a `string` type variable.

@kushaldas, can you check this?

### CODE REVIEW

File: bugspad.go [hunk 1]
```diff
@@ -134,6 +134,30 @@ func components(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func get_bug_details(w http.ResponseWriter, r *http.Request) {
```
Comment (kushaldas) [inline bugspad.go:137]: This should be handles in the GET method of /bug/ URL.

File: bugspad.go [hunk 2]
```diff
@@ -134,6 +134,30 @@ func components(w http.ResponseWriter, r *http.Request) {
...
+    var bug_id string
```
Comment (kushaldas) [inline bugspad.go:138]: You can directly use title for the same.

**Comment by kushaldas** [comment]
Actually I also wrote same thing in my git branch, not yet pushed, I am ok to push this but make the necessary changes.

**Comment by rtnpro** [comment]
@kushaldas what about the description field, I am not able to fetch it's value from the db. It always comes as an empty string.

**Comment by kushaldas** [comment]
I just now pushed my code, which does this properly, test and tell me.

**Pull request closed without merging**